content/panorama/scripts/custom_game/time_control.ts

```diff
@@ -0,0 +1,26 @@
+function setTimerValue(event:ClockTimeEvent) {
+    var mainPanel = $.GetContextPanel().GetParent()!.GetParent()!;
+    var gameTime = mainPanel.FindChildTraverse("GameTime") as LabelPanel;
+
+    $.Msg(  event.seconds,event.minutes)
```

Comment:
What is this formatting, also I guess this was for debugging and is no longer needed?

Reply:
Not sure what you mean with the formatting